src/hooks/useMintVolume.ts

```diff
@@ -1,8 +1,6 @@
-import useSWR from "swr";
 import moment from "moment";
 import axios from "axios";
```

Comment:
it's because I think we just have to call `axios` until the `result` state is completed from the first until the latest block number and it doesn't have to do a real-time update because the API call is limited to 5 per second.  However, this hooks is actually just a temporary one because we are gonna migrating it to [https://thegraph.com/hosted-service/](https://thegraph.com/hosted-service/) and refactor it ASAP.

Reply:
so actually it's a while loop inside the `useEffect` because we can't query all the data. So it's chunk by chunk, looping until it's completed